sigle/src/modules/editor/components/SlateEditor.tsx

```diff
@@ -184,6 +181,7 @@ export const SlateEditor = ({
       ? Value.fromJSON(story.content)
       : Value.fromJSON(emptyNode as any)
   );
+  const { user } = useAuth();
```

Comment:
In general for the code style, let's try to put the hooks on top (before the `useState` and `useRef`) :)